SpywareFilter/sections/specific.txt

```diff
@@ -23,6 +23,26 @@
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
+! realty.chosun.com - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||news.chosun.com/svc/fb_count.html
+||news.chosun.com/dhtm/*/collecter.js
+! w.atwiki.jp - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||w.atwiki.jp/*/google_analytics.js
+||w.atwiki.jp/*/counter.js
+! ytn.co.kr - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||ats.auction.co.kr/cpc/imp/nw?r
+||engine.tend-table.com/cgi-bin/WebLog.dll
+||grenade.ebaykorea.com/api/getImps
```

Comment:
Not reproduced:
```
||ats.auction.co.kr/cpc/imp/nw?r
||engine.tend-table.com/cgi-bin/WebLog.dll
||grenade.ebaykorea.com/api/getImps
```

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b77b7616a50c637661f561a74c6208eeffc7d547